installer/create-database.php

```diff
@@ -396,7 +396,7 @@ function createDatabase($oDB){
         $oDB->createCommand()->createTable('{{sessions}}', array(
             'id' => "string(32) NOT NULL",
             'expire' => "integer NULL",
-            'data' => "binary",
+            'data' => "string(255) NULL",
```

Comment:
That can't be right. Session data is huge and cannot fit into such short field type.

Reply:
Since it totally broke on MSSQL : why not just an text (or ntext) ?